### ISSUE TITLE
cannot install requirements

### ISSUE THREAD

**Issue by iwakgatul** [enarjord/passivbot#133]
this is what i got

Could not find a version that satisfies the requirement websockets>=8.1 (from -r requirements.txt (line 1)) (from versions: 1.0, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 3.0, 3.1, 3.2, 3.3, 3.4, 4.0.1, 5.0, 5.0.1)
No matching distribution found for websockets>=8.1 (from -r requirements.txt (line 1))

i change python version manytimes, but no progress at my side
somebody help me out pls. T_T
im using Ubuntu 18.04.5 LTS

**Comment by iwakgatul** [comment]
python 3.8.1 sir

**Comment by XMVZ** [comment]
Yeah this is not working for me either... Please help I'd really like to be able to use the bot.

**Comment by ghostnetrn** [comment]
sudo apt install -y python3-pip
sudo apt install -y build-essential libssl-dev libffi-dev python3-dev

**Comment by XMVZ** [comment]
> sudo apt install -y python3-pip
> sudo apt install -y build-essential libssl-dev libffi-dev python3-dev

and on windows?

**Comment by ghostnetrn** [comment]
pip install python-dateutil --upgrade
python -m pip install --upgrade pip

**Comment by ghostnetrn** [comment]
Try too https://phoenixnap.com/kb/install-pip-windows

**Comment by XMVZ** [comment]
> Try too https://phoenixnap.com/kb/install-pip-windows

Seems to be an issue with Websockets

**Comment by XMVZ** [comment]
Tried on Linux. Got to the same point. No module named Websockets. 

Shame, I really want to use this!

**Comment by eliafino** [comment]
Minimum Python 3.8.3. Compile from source

**Comment by XMVZ** [comment]
> Minimum Python 3.8.3. Compile from source

Have gotten to this point now...
[LINK](https://gyazo.com/a0e4607f305e96944b8abd5d9708f4ff)

**Comment by eliafino** [comment]
@mxwo93 seems to work correctly

**Comment by XMVZ** [comment]
@eliafino what am I missing? why is it saying connection with websocket lost?
i am fairly new to this - how do i get it to start actively making trades instead of returning errors like the one i just shared?

**Comment by XMVZ** [comment]
[LINK](https://gyazo.com/516b8a52f6e8ca86613edfca27c221a6)
to me.. this does not seem to be working correctly. @eliafino

**Comment by eliafino** [comment]
@mxwo93 Please remove manually /home/max/.passivbotlock

**Comment by XMVZ** [comment]
@eliafino hope you're well.
was wondering if you knew how to change position size. can't seem to find variable in config or in telegram bot.

let me know if you can, thank you!

**Comment by eliafino** [comment]
3.5 or 3.6 bot version? Entry qty position? Come in Discord or telegram!

**Comment by robispot** [comment]
spent for few hours, try install using Python 3.10.14 works for me
